src/components/AudioCell.vue

```diff
@@ -121,6 +131,80 @@ const audioItem = computed(() => store.state.audioItems[props.audioKey]);
 
 const uiLocked = computed(() => store.getters.UI_LOCKED);
 
+const isMultiSelectEnabled = computed(
+  () => store.state.experimentalSetting.enableMultiSelect
+);
+
+const setActiveAudioKey = () => {
+  if (store.getters.ACTIVE_AUDIO_KEY !== props.audioKey) {
+    store.dispatch("SET_ACTIVE_AUDIO_KEY", { audioKey: props.audioKey });
+    store.dispatch("SET_SELECTED_AUDIO_KEYS", { audioKeys: [props.audioKey] });
+  }
+};
```

Comment:
関数名とやっていることが異なりそうです。
やっていることは「1つ選択し、アクティブにする」なので、。それを名称にするのはどうでしょう？

あとそれがアクティブだった場合に選択の挙動が変わるのって結構違和感がありそうです（ユーザーの感心的に選択したいかどうかとアクティブであるかどうかが関係がないことが多そうなので）。
これに関しては議論の余地があるかもしれません。でも何にせよセッターの中に条件式が入っているのは違和感があるので、書く場合は関数名を注意した方がいいかも？
```suggestion
const selectAndActiveAudioKey = () => {
  store.dispatch("SET_ACTIVE_AUDIO_KEY", { audioKey: props.audioKey });
  store.dispatch("SET_SELECTED_AUDIO_KEYS", { audioKeys: [props.audioKey] });
};
```

Reply:
> ユーザーの感心的に選択したいかどうかとアクティブであるかどうかが関係がないことが多そうなので

このsetActiveAudioKey、ウィンドウ切り換えて戻すときのfocusでも発火しちゃうんですよね...そのため保護してます。focusの方に移動します

Reply:
なるほどです････････。
どちらもバグ的な挙動なので、両方とも直せるのを目指すのが良さそうですかね･･･。

そもそも複数選択されている状態でinputにフォーカスが当たってる状態をなくせば良さそうに思いました。
今複数選択とフォーカスが同時に発生しうるのは、おそらくshift+上下キーだけだと思います。
この操作の時にフォーカスを与えないようにするのはどうでしょう･････？

そして複数選択とフォーカスされている状態は同時に起こらないようにすることをどこかにメモっておけば一旦良さそうかなと！！！

<details>
<summary>ユーザーニーズについての感想</summary>

アクティブなものと非アクティブなもので、表示以外の挙動を変えるのは避けるべきだと思う。アクティブであるかどうかに関係ないはずの操作が阻害されるのはユーザーにとって普通にストレスなはず。

ウィンドウのフォーカスが戻ってきた時にinputにフォーカスが当たり直すのは便利なのでそのままにしておきたい。ただしどちらかというと上述のものよりは発生件数がレアなので、二者択一で犠牲が必要なのであれば、多分こっちを犠牲にすることになる。

複数選択とフォーカスが当たってる状態の同時発生は多分必要ないはず。あるとしたらテキストを一気に変更したい時ぐらい。なのでややこしい状態として存在不可にできそう。

</details>

Reply:
複数選択をしてる状態の時にエンターキーを押すと、アクティブなテキスト欄に対してフォーカスが当たることに気づきました。。。タブ移動した時も多分フォーカスされますね。。
これに関しては、若干苦渋の選択ですが、フォーカスが当たったタイミングで複数選択を解除するのがいいのかなと思いました。。

Reply:
> そもそも複数選択されている状態でinputにフォーカスが当たってる状態をなくせば解決しそうに思いました！

かなり複雑になりそうなんですよね…

> アクティブなものと非アクティブなもので、表示以外の挙動を変えるのは避けるべきだと思う。アクティブであるかどうかに関係ないはずの操作が阻害されるのはユーザーにとって普通にストレスなはず。

Ctrl/Shiftなしの`@click`生やしてその時は`selected = [active]`みたいな処理にすればユーザー目線では処理が変わらないと思います。

> あと複数選択中に非アクティブなキャラクターアイコンをクリックした時にアクティブが移動して選択状態が解除されますが、これはそもそもアクティブが移動しないようにすると良さそう。

これは確かに。複数選択時は移動しないようにしました。

Reply:
> Ctrl/Shiftなしの@click生やしてその時はselected = [active]みたいな処理にすればユーザー目線では処理が変わらないと思います。

複数選択用のdivの`@click`を追加して、選択されたらAudioCellのinputをactiveにする、という意味で合ってますか？
そのまま実装するとAudioCellのボタンが押せなくなる気がします 😇 

なので↓がとりあえずの妥協案かなぁと･･･

> これに関しては、若干苦渋の選択ですが、フォーカスが当たったタイミングで複数選択を解除するのがいいのかなと思いました。。

Reply:
> 複数選択用のdivの@clickを追加して、選択されたらAudioCellのinputをactiveにする、という意味で合ってますか？
そのまま実装するとAudioCellのボタンが押せなくなる気がします 😇

あ、AudioCellの一番上のdivですね。stopPropagationしなければ押せると思います

Reply:
activeにした直後に消去も発動って感じになりますかね。画面が瞬時に切り替わるので避けたいですが、そうなるとStopPropagationするかしないかが下にあるViewによって変わるという実装になる…？
（だいぶ複雑になりそう）

他にもキャラボタン、input、消去ボタン、ホバー状態による消去ボタン表示非表示、ドラッグによる移動、複数選択、右クリック等いろいろあるので、これらがいけるかどうかってとこですかね。。。

他の手として、もう挙動がわからないinput使うのやめて、入力可能なspanにし、全てのイベントやアクセシビリティを自前実装するのもなくはないかなと思います。
VSCodeが使ってるテキストインプットのライブラリ（名前忘れました。。）はそんな感じなので、それ流用するとかもありかも？

まあとりあえずfocus=選択でどうでしょう…。いうて選択解除されるだけなはず。

Reply:
多分想定通りの動作に出来たと思います。

>  VSCodeが使ってるテキストインプットのライブラリ（名前忘れました。。）

もしかして：Monaco Editor？

Reply:
あ、ですです！！
一旦ここでの議論は解決だと思うのでresolveします、ありがとうございました！！